android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/AdjustKeyboardHeightActivity.java

```diff
@@ -77,9 +73,7 @@ protected void onCreate(Bundle savedInstanceState) {
     resetButton.setOnClickListener(new View.OnClickListener() {
       public void onClick(View v) {
         // Clear the keyboard height preferences to reset
-        editor.remove(KMManager.KMKey_KeyboardHeightPortrait);
-        editor.remove(KMManager.KMKey_KeyboardHeightLandscape);
-        editor.commit();
+        KMManager.applyKeyboardHeight(this, 0);
```

Comment:
You can use `context` assigned in l.44

```suggestion
        KMManager.applyKeyboardHeight(context, 0);
```